app/views/facility_accounts/_account_balances.html.haml

```diff
@@ -7,14 +7,13 @@
       %tr
         %th= t(".th.account")
         %th= t(".th.owner")
-        %th.currency= t(".th.total")
+        %th= t(".th.expiration")
```

Comment:
Now that this file doesn't include balances anymore, should we change the filename?

Also, should we use `Account.human_attribute_name(:expires_at)` to keep this header consistent?